qemu/tests/cfg/multi_host.cfg

```diff
@@ -37,15 +37,20 @@
                 #Time when start migration
                 - after_login_vm:
                     paused_after_start_vm = no
+                    need_to_login = yes
+                    preprocess_env = no
                 - early_boot_vm:
                     no measure_migration_speed
                     login_timeout = 420
                     paused_after_start_vm = yes
+                    preprocess_env = yes
                     variants:
                         - timeout_0:
                             start_migration_timeout = 0
                         - timeout_6:
                             start_migration_timeout = 6
+                        - timeout_random:
+                            start_migration_timeout = random
```

Comment:
do it in virttest/utils_test/qemu.py:
@@ -731,6 +735,8 @@ class MultihostMigration(object):
                         for vm in mig_data.vms:
                             vm.resume()
                         wait = self.params.get("start_migration_timeout", 0)
+                        if wait == "random":
+                            wait = random.randint(5, 20)